packages/@expo/prebuild-config/CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
 
 ### 🐛 Bug fixes
 
+- [iOS] remove default APNS entitlement. ([#27924](https://github.com/expo/expo/pull/27924) by [@douglowder](https://github.com/douglowder))
```

Comment:
This is a breaking change. If a third-party package depended on apns to be added, now it cannot.

Reply:
Fixed